R/plot-population-time-profile.R

```diff
@@ -23,21 +21,27 @@
 #' myDataComb <- DataCombined$new()
 #' myDataComb$addSimulationResults(populationResults)
 #'
-#' # Create a new instance of `DefaultPlotConfiguration` class
-#' myPlotConfiguration <- DefaultPlotConfiguration$new()
-#' myPlotConfiguration$title <- "My Plot Title"
-#' myPlotConfiguration$subtitle <- "My Plot Subtitle"
-#' myPlotConfiguration$caption <- "My Sources"
 #'
 #' # plot
-#' plotPopulationTimeProfile(myDataComb, myPlotConfiguration)
+#' plotPopulationTimeProfile(myDataComb)
+#'
+#' # plot with other quantiles
+#' plotPopulationTimeProfile(myDataComb, probs = c(0.1, 0.5, 0.9))
+#'
+#' # plot with arithmetic mean
+#' plotPopulationTimeProfile(myDataComb,
+#'                           aggregation = "arithmetic")
+#
+#' # plot with arithmetic mean and 3sd above and below average line
+#' plotPopulationTimeProfile(myDataComb,
+#'                           aggregation = "arithmetic",
+#'                           n=3)
 #'
 #' @export
 plotPopulationTimeProfile <- function(dataCombined,
```

Comment:
I really do hate the `...` in public functions and would like to avoid it as much as possible. In this case, let's get the `quantiles` argument back. Otherwise, this would be a breaking change which is not necessary here.

Also, the `n` argument is a bit confusing, am I not aware of use cases where someone would want to plot n-times sd instead of `sd` only.